ray-operator/DEVELOPMENT.md

```diff
@@ -144,7 +145,7 @@ make manifests
 make helm
 
 # Synchronize 1, 2, 3, and 4 in one command
-# [Note]: Currently, we need to synchronize consistency 5 manually.
+# [Note]: Currently, we need to synchronize consistency 5 and 6 manually.
```

Comment:
Does CI currently catch it if someone forgets to do this?  If not, maybe we should create an issue to track automating this, otherwise there's a risk of someone breaking it accidentally

Reply:
I see, you've included it as a "follow up"